apps/backend/src/parse/entities/program-daily-upload.entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Column,
+  CreateDateColumn,
+  PrimaryGeneratedColumn,
+  Entity,
+  OneToMany,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm';
+import { FileUploadedEntity } from './file-uploaded.entity';
+import { FileIngestionRulesEntity } from './file-ingestion-rules.entity';
+
+@Entity('program_daily_upload')
+export class ProgramDailyUploadEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @CreateDateColumn()
+  created_at: Date;
+
+  @Column({ type: 'date' })
```

Comment:
I *think* that typeorm returns the postgres "date" as a string, rather than the date